ffwd-reporter/src/main/java/com/spotify/metrics/ffwd/FastForwardReporter.java

```diff
@@ -396,4 +338,101 @@ public void stop() {
     public void close() {
         stop();
     }
+
+    public static final class Builder {
```

Comment:
It'd probably be better to make separate PRs for changes like reordering the structure of the class or whitespace changes, as it makes the git history harder to follow if its combined with this PR

Reply:
+1 to this comment - pls separate this into diff PR

Reply:
Done